daemon/policy.go

```diff
@@ -274,56 +274,8 @@ func (d *Daemon) PolicyDelete(labels labels.LabelArray) (uint64, error) {
 		return rev, apierror.New(DeletePolicyNotFoundCode, "policy not found")
 	}
 
-	go func() {
```

Comment:
Should we update the Header comment. It looks outdated
```// PolicyDelete deletes the policy set in the given path from the policy tree.
 // If cover256Sum is set it finds the rule with the respective coverage that
 // rule from the node. If the path's node becomes ruleless it is removed from
 // the tree.

Reply:
Really outdated, fixed! :-)